tasks/preflight.yml

```diff
@@ -31,6 +31,7 @@
   stat:
     path: "/usr/local/bin/node_exporter"
   register: __node_exporter_is_installed
+  check_mode: no
```

Comment:
yamllint expects `false` as a value and won't accept `no`. Please change to `false`.